kickoff/model.py

```diff
@@ -281,24 +316,12 @@ def getReleases(ready=None, complete=None, shipped=None, productFilter=None,
             qry = table.query.filter_by(**filters)
             if shipped:
                 qry = qry.filter(table._shippedAt != None)
-            if lastRelease and not paginationCriteria:
+            if lastRelease:
                 # Sort using version
                 results = sorted(
                     qry.all(), key=lambda x: LooseVersion(x.version),
                     reverse=True)
             else:
-                if searchFilter:
-                    searchList = table.getSearchList(searchFilter)
-                    qry = qry.filter(or_(*searchList))
-
-                if paginationCriteria:
-                    if productFilter:
-                        qry = qry.filter(ProductReleasesView.product.contains(productFilter))
-
-                    orderByList = getOrderByList(table, paginationCriteria.orderByDict)
-
-                    qry = qry.order_by(*orderByList).limit(paginationCriteria.length).offset(paginationCriteria.start)
-
```

Comment:
Yeah, ProductReleasesView has no more "table behavior". Reviewing better now, the ProductReleasesView should never be here haha.